### PR TITLE
Add PR checks for the Control branch

### PR DESCRIPTION
**Pull request by DaveTryon**
#### Describe the change
We want automated checks to validate that proposed changes to the Control branch will not result in breaking user upgrade scenarios. To this end, add a prbuild-Control.yml file that triggers a PowerShell script to validate the contents of the **Insider** and **Production** _release_info.json_ files. This script has been debugged using a branch off the Control branch, and is able to catch the following errors within the _release_info.json_ files:
- The _release_info.json_ file is missing
- The **default** section is missing
- Any of _current_version_, _minimum_version_, _installer_url_, or _release_notes_url_ is missing
- The _current_version_ or _minimum_version_ values can't be parsed into System.Version objects
- The _current_version_ or _minimum_version_ values reference tags that do not exist
- The value of _current_version_ >= _minimum_version>
- The _installer_url_ or _release_notes_url_ values point to objects that can't be downloaded

Once this is merged, I'll complete the changes to trigger the script as part of every PR against the **Control** branch.

#### PR checklist

- [ ] Run through of all [test scenarios](https://github.com/Microsoft/accessibility-insights-windows/blob/master/docs/Scenarios.md) completed?
- [ ] Does this address an existing issue? If yes, Issue# - 
- [ ] Includes UI changes?
  - [ ] Run the production version of Accessibility Insights for Windows against a version with changes.
  - [ ] Attach any screenshots / GIF's that are applicable.

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

File: tools/scripts/ValidateReleaseInfo.ps1 [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+	WriteDivider
+	return $isCurrentVersionValid -and $isMinimumVersionValid -and $isInstallerUrlValid -and $isReleaseNotesUrlValid
+}
+
+# Load the octokit dll
+Add-Type -Path ((Get-Location).Path + '\Octokit.0.32.0\lib\net45\Octokit.dll')
+
+# Get a new client with an appropriate product header value
+$productHeader = [Octokit.ProductHeaderValue]::new("AIWindows-ReleaseInfoValidation")
+$client = [Octokit.GitHubClient]::new($productHeader)
```
Comment (DaveTryon) [inline tools/scripts/ValidateReleaseInfo.ps1]: No credential is needed to read the available releases, so there's no worry about accidentally leaking a secret if someone submits a PR

File: tools/scripts/ValidateReleaseInfo.ps1 [hunk 2]
```diff
@@ -0,0 +1,98 @@
+# Constants
```
Comment (DaveTryon) [inline tools/scripts/ValidateReleaseInfo.ps1]: Actual output of the script that appears in the PR logs:

Existing Releases
v1.1.0830.03
v1.1.0830.01
v1.1.0828.02
v1.1.0828.01
v1.1.0827.02
v1.1.0827.01
v1.1.0823.01
v1.1.0822.06
v1.1.0794.01
-----------------------
Checking Info for Production
Channel Info: @{current_version=1.1.0794.01; minimum_version=1.1.0794.01; installer_url=https://www.github.com/Microsoft/accessibility-insights-windows/releases/download/v1.1.0794.01/AccessibilityInsights.msi; release_notes_url=https://www.github.com/Microsoft/accessibility-insights-windows/blob/Control/Channels/Production/release_notes.md}
PASSED Does release v1.1.0794.01 exist?
PASSED Does release v1.1.0794.01 exist?
PASSED Does web resource https://www.github.com/Microsoft/accessibility-insights-windows/releases/download/v1.1.0794.01/AccessibilityInsights.msi exist?
PASSED Does web resource https://www.github.com/Microsoft/accessibility-insights-windows/blob/Control/Channels/Production/release_notes.md exist?
-----------------------
Checking Info for Insider
Channel Info: @{current_version=1.1.0828.01; minimum_version=1.1.0828.01; installer_url=https://www.github.com/Microsoft/accessibility-insights-windows/releases/download/v1.1.0828.01/AccessibilityInsights.msi; release_notes_url=https://www.github.com/Microsoft/accessibility-insights-windows/blob/Control/Channels/Insider/release_notes.md}
PASSED Does release v1.1.0828.01 exist?
PASSED Does release v1.1.0828.01 exist?
PASSED Does web resource https://www.github.com/Microsoft/accessibility-insights-windows/releases/download/v1.1.0828.01/AccessibilityInsights.msi exist?
PASSED Does web resource https://www.github.com/Microsoft/accessibility-insights-windows/blob/Control/Channels/Insider/release_notes.md exist?
-----------------------
Successfully validated release_info.json files

**Comment by karanbirsingh** [comment]
How do you feel about also checking whether the current version is >= the minimum version?

**Comment by DaveTryon** [comment]
Good idea. I'll add this to the script

---
In reply to: [482639451](https://github.com/Microsoft/accessibility-insights-windows/pull/293#issuecomment-482639451) [](ancestors = 482639451)

**Comment by DaveTryon** [comment]
Added

---
In reply to: [482643799](https://github.com/Microsoft/accessibility-insights-windows/pull/293#issuecomment-482643799) [](ancestors = 482643799,482639451)

File: tools/scripts/ValidateReleaseInfo.ps1 [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+$repositoryName = 'accessibility-insights-windows'
+
+$pathToRepo = $($args[0])
+
+function WriteDivider()
+{
+	Write-Host '-----------------------'
+}
+
+function GetStatus($trueOrFalse)
```
Comment (AhmedAbdoOrtiga) [inline tools/scripts/ValidateReleaseInfo.ps1]: should we change thew parameter name to status or valid

**Pull request merged**